app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -68,6 +98,45 @@ export default {
     }
   },
   methods: {
+    // submit a delegation
+    async onStake({ amount }) {
+      const to = this.validator.owner
+
+      const currentlyDelegated =
+        parseInt(this.delegation.committedDelegates[to]) || 0
+
+      const delegation = [
+        {
+          atoms: currentlyDelegated + amount,
+          delegate: { owner: to }
+        }
+      ]
+
+      try {
+        await this.$store.dispatch("submitDelegation", delegation)
+
+        this.$store.commit("notify", {
+          title: "Successful Staking!",
+          body: `You have successfully staked your ${this.bondingDenom}s.`
+        })
+      } catch ({ message }) {
+        let errData = message.split("\n")[5]
```

Comment:
is this from @fedekunze's error catch PR from a while ago? Wasn't this supposed to be handled by the SDK eventually?

Reply:
This is copied-and-pasted code from PageBond.vue.  I don't like the code but felt that fixing it was out of scope for this PR.